ru/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -28,51 +28,51 @@
 <!ENTITY closeCmd.accesskey                             "C">
 <!ENTITY printSetupCmd.label                            "Параметры страницы…">
 <!ENTITY printSetupCmd.accesskey                        "u">
-<!ENTITY printPreviewCardViewCmd.label                  "Print Preview Card">
+<!ENTITY printPreviewCardViewCmd.label                  "Предварительный просмотр карточки">
 <!ENTITY printPreviewCardViewCmd.accesskey              "v">
 <!ENTITY printPreviewAddressBook.label                  "Предварительный просмотр адресной книги">
 <!ENTITY printPreviewAddressBook.accesskey              "B">
-<!ENTITY printCardViewCmd.label                         "Print Card...">
-<!ENTITY printCardViewCmd.accesskey                     "P">
+<!ENTITY printCardViewCmd.label                         "Печать карточки…">
+<!ENTITY printCardViewCmd.accesskey                     "p">
```

Comment:
OK, sorry. 
I have reverted this hotkey change. Should I commit the fixed file again?